pkg/server/cluster_server.go

```diff
@@ -13,7 +13,7 @@ import (
 	clientcmd "k8s.io/client-go/tools/clientcmd"
 	clientcmdv1 "k8s.io/client-go/tools/clientcmd/api/v1"
 
-	v1 "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned/typed/machineconfiguration.openshift.io/v1"
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned/typed/machineconfiguration.openshift.io/v1"
```

Comment:
…/pkg/generated/… is a different package; should it use the same prefix?

Reply:
maybe this is the distinction we should make here, call the generated pkg (and only that!) `v1`
Will revert this part.